src/SFA.DAS.AANHub.Application.UnitTests/Partners/Queries/GetPartnerMemberQueryHandlerTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Partners.Queries;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Partners.Queries
+{
+    [TestFixture]
+    public class GetPartnerMemberQueryHandlerTests
+    {
+
+        [Test]
+        public async Task Handle_GetPartnerMember()
+        {
+            string userName = "";
+            Guid memberid = new();
+
+            var partnersReadRepositoryMock = new Mock<IPartnersReadRepository>();
+            var partner = new Partner();
+            partnersReadRepositoryMock.Setup(a => a.GetPartnerByUserName(userName)).ReturnsAsync(partner);
+            var sut = new GetPartnerMemberQueryHandler(partnersReadRepositoryMock.Object);
+
+            var result = await sut.Handle(new GetPartnerMemberQuery(userName), new CancellationToken());
+
+            Assert.AreEqual(memberid, result!.Result.MemberId);
```

Comment:
Probably should just check that the other properties are populated as expected

Reply:
Added other properties. Thanks for this.